pkg/volume/rbd/rbd.go

```diff
@@ -486,16 +486,29 @@ func (c *rbdUnmounter) TearDownAt(dir string) error {
 	return diskTearDown(c.manager, *c, dir, c.mounter)
 }
 
-func getVolumeSource(
-	spec *volume.Spec) (*v1.RBDVolumeSource, bool, error) {
+func getVolumeSource(spec *volume.Spec) ([]string, string, string, string, string, string, bool, error) {
```

Comment:
this many return values of identical types is asking for trouble... individual accessors are simple and clear

Reply:
done